book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -604,6 +614,8 @@ Committing a block to the sled state should be implemented as a wrapper around
 a function also called by [`Request::CommitBlock`](#request-commit-block),
 which should:
 
+### `pub(super) fn commit_finalized_block(&mut self, block: QueuedBlock) -> block::Hash`
+
```

Comment:
This was already added in my PR based on your suggestion, would you mind changing the base commit of this PR to be the `chainset-impl` branch so you're working with the most up to date copy?

Reply:
We merged #1011 into main, then I rebased on main.